changelog.html

```diff
@@ -24,7 +24,7 @@
 <!--[if IE]>
 <style type="text/css">div.rate-offset { bottom: 0.2em !important; left: 5em !important; }</style>
 <![endif]-->
-  <script type="text/javascript" src="/rate/rate.js"></script>
+  <script type="text/javascript" src="https://jenkins-ci.org/rate/rate.js"></script>
```

Comment:
@oleg-nenashev Take that up with @rtyler, AFAIK all services besides the site and the account app are still on the other domain. There is no https://jenkins.io/rate/rate.js.